src/lib.rs

```diff
@@ -45,6 +72,34 @@ pub struct BMFont {
 }
 
 impl BMFont {
+    /// Constructs a new [BMFont].
+    ///
+    /// ## Examples
+    ///
+    /// From a file:
+    ///
+    /// ```rust
+    /// # use bmfont::*;
```

Comment:
Why all these comments? Here and below.

Reply:
I was following [this guidance](https://doc.rust-lang.org/rustdoc/what-to-include.html) about hiding boilerplate stuff.

In my library I've put a really detailed doc example at the library root in the ///! section and then I'm trying to make the other examples not show that stuff. YMMV, this depends on how you'd like to document this crate.

In the larger sense, the comments are there because this code gets compiled and tested with the `cargo test` command.